extern-sdk-swagger-adaptor/src/main/java/ru/kontur/extern_api/sdk/service/transport/adaptor/swagger/EventsAdaptorImpl.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */

```

Comment:
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */